docs/k8s-ibm-public.md

```diff
@@ -97,12 +97,15 @@ whisk:
 
 k8s:
   persistence:
-    defaultStorageClass: default
+    hasDefaultStorageClass: false
+    explicitStorageClass: default
 ```
```

Comment:
this doesn't work for IKS 10.08. I don't understand why. K8s 10.08 is also a stable version. Do you think we want to support 10.08?

Reply:
I believe IKS v10.08 does support dynamic provisioning. It may need additional configuration, which is hidden somewhere. Yet I cannot figure out.